src/datepicker.android.js

```diff
@@ -22,6 +22,7 @@ export default class DatePickerAndroid {
    *   - `value` (`Date` object) - date to show by default
    *   - `minimumDate` (`Date` object) - minimum date that can be selected
    *   - `maximumDate` (`Date` object) - maximum date that can be selected
+   *   - `testID` (`string`) - testID for usage with e.g. detox
```

Comment:
I guess it is called **accessibility tags** on Android.

![](https://wix.github.io/Detox/assets/images/tagAttributeAndroid-2c0ded43f52dee82ba0f4f928c90385d.png)

I'd suggest to be less specific and replace it with "automation frameworks".

Reply:
@noomorph I just rephrased it. What do you think about it now?